src/Storage/appsettings.json

```
@@ -1,4 +1,11 @@
 {
+  "PostgreSQLSettings": {
+    "WorkspacePath": "Migration",
+    "AdminConnectionString": "Host=localhost;Port=5432;Username=platform_storage_admin;Password={0};Database=storagedb",
+    "ConnectionString": "Host=localhost;Port=5432;Username=platform_storage;Password={0};Database=storagedb;Include Error Detail=True",
+    "StorageDbAdminPwd": "<get-from-key-vault>",
+    "StorageDbPwd": "<get-from-key-vault>"
```

```suggestion
    "StorageDbPwd": "Password"
```

let's include default password to simplify local development. No KV involved then.